ptf_tests/common/utils/test_utils.py

```diff
@@ -228,7 +228,29 @@ def vm_to_vm_ping_test(conn, dst_ip, count="4"):
     else:
         print(f"FAIL: Ping Failed to destination {dst_ip} with {pkt_loss}% loss")
         return False
-    
+
+def vm_to_link_port_ping_drop_test(conn, dst_ip, count="4"):
```

Comment:
there is already one util function  vm_to_vm_ping_drop_test .... is this doing anything different?